flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
+    ]
+}
```

Comment:
arm64 is available https://file-assets.apifox.com/download/Apifox-linux-arm64-latest.zip https://file-assets.apifox.com/download/Apifox-linux-arm64-deb-latest.zip, why not use it?